.github/workflows/ci-build.yml

```diff
@@ -9,14 +10,27 @@ on:
 jobs:
   lint-and-unit-test:
     name: Lint & Unit Tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout repo
         uses: actions/checkout@v2
+      - name: Add apt repo
+        run: sudo add-apt-repository universe
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+          java-package: jdk
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+          architecture: x64
```

Comment:
With your last comment in mind, @MRichards99, we can probably also remove these lines too. This takes us back to what is currently on master, which works fine. There's no need for the universe repo, java or python in these steps.